docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3'
+
+services:
+  base:
+    build: ./
+    image: "ryanthemanuel/terra-core"
+  standalone-chrome:
+    image: 'selenium/standalone-chrome:3.11.0-californium'
```

Comment:
Is toolkit locking this down? I know we talked about toolkit doing that? If so, is there a way to ensure we always reference that same version?